dtool/src/dtoolutil/textEncoder.h

```diff
@@ -141,6 +141,7 @@ class EXPCL_DTOOL_DTOOLUTIL TextEncoder {
   static Encoding _default_encoding;
 };
 
+
```

Comment:
PR should avoid any edits to `textEncoder.h` at all if it's not changing anything of substance. Could you delete this extra newline?